software/CONFIGURATION.md

```diff
@@ -22,9 +22,10 @@ default configuration:
 }
 ```
 
-- `EUROPI_MODEL` specifies the type of EuroPi module. Currently only `"europi"` is supported. Default: `"europi"`
-- `PICO_MODEL` must be one of `"pico"` or `"pico w"`. Default: `"pico"`
-- `CPU_FREQ` must be one of `250000000` or `125000000`. Default: `"250000000"`
+- `EUROPI_MODEL` specifies the type of EuroPi module. Currently only `"europi"` is supported. Default: `"europi"`.
+- `PICO_MODEL` must be one of `"pico"`, `"pico2"`, or `"pico w"`. Default: `"pico"`.
```

Comment:
[Suggestion]
Not technically within this PR's scope but potentially worth adding `"pico h"` as an alternate name for `"pico"` in case beginners have bought it not realising it's identical in all but header assembly